src/components/inline/DonationProjectCard/DonationProjectCard.stories.js

```diff
@@ -0,0 +1,23 @@
+import React from "react";
+import DonationProjectCard from "./DonationProjectCard";
+import background from "./blackbackground.jpg";
+
+export default {
+  title: "Inline/DonationProjectCard",
```

Comment:
I remember in @cw323 card commit that @richardxia  wanted to move the cards into the block directory. So the file path should be src/block/WhateverComponentHere. Then this should change to Block/DonationProjectCard.

Reply:
👍 The Block folder created in the component folder separate from the inline folder. Title changed to "title: "Block/DonationProjectCard"".